### PR TITLE
Reframe Buttons and Cards project

### PR DESCRIPTION
**Pull request by timkinnane**
@diegodorgam please review to see if it's still inline with your idea of the project. I've tried to keep the requirements quite specific but expand the vision, to hopefully be more attractive to students.

### CODE REVIEW

File: contributing/google-summer-of-code-2018/README.md
Comment (diegodorgam) [inline contributing/google-summer-of-code-2018/README.md]: I think we should keep buttons and cards reference in the title. Like `Rich Messages and Payloads for Buttons and Cards`

File: contributing/google-summer-of-code-2018/README.md
Comment (diegodorgam) [inline contributing/google-summer-of-code-2018/README.md:182]: awesome, much better! thanks @timkinnane

**Review by MartinSchoeler — APPROVED** [review]

**Comment by MartinSchoeler** [comment]
Waiting on @diegodorgam approval to merge

File: contributing/google-summer-of-code-2018/README.md
Comment (timkinnane) [reply contributing/google-summer-of-code-2018/README.md]: @diegodorgam sorry looks good to me and I've been meaning to action this but I just saw it got pulled without that. Should we do another PR to change it as you suggested?

**Pull request merged**
